docs/reference/content/guides/annotations.md

```diff
@@ -0,0 +1,292 @@
+```java
+public @interface IndexOptions {
+    String name() default "";
+    boolean unique() default false;
+    boolean dropDups() default false;
+    boolean background() default false;
+    boolean sparse() default false;
+    boolean disableValidation() default false;
+    int expireAfterSeconds() default -1;
+    String language() default "";
+    String languageOverride() default "";
+}
+```
+| Parameter           | Usage           |
+| ------------------- | --------------  |
+| name() | The name of the index |
+| unique() | Requires values in the index to be unique |
+| dropDups() | Drop any duplicate values during the creation of a unique index. |
+| background() | Create this index in the background.  There are some [considerations](http://docs.mongodb.org/manual/tutorial/build-indexes-in-the-background/#considerations) to keep in mind.
+| sparse() | Create a [sparse](http://docs.mongodb.org/manual/core/index-sparse/) index |
+| expireAfterSeconds() | Creates a [TTL Index](http://docs.mongodb.org/manual/core/index-ttl/) on a date field. |
+| disableValidation() | By default, Morphia will validate field names being index.  This disables those checks. |
+| language() | Default language for the index. |
+| languageOverride() | The field in the document to use to override the default language. |
+
+#### Indexed
+Applied to a Java field, marks the field to be indexed by mongodb.
+
+```java
+public @interface Indexed {
+    IndexDirection value() default IndexDirection.ASC;
+    String name() default "";
+    boolean unique() default false;
+    boolean dropDups() default false;
+    boolean background() default false;
+    boolean sparse() default false;
+    int expireAfterSeconds() default -1;
+}
+```
+| Parameter           | Usage           |
+| ------------------- | --------------  |
+| value() | The sort direction for the index |
+| name() | The name of the index |
+| unique() | Requires values in the index to be unique |
+| dropDups() | Drop any duplicate values during the creation of a unique index. |
+| background() | Create this index in the background.  There are some [considerations](http://docs.mongodb.org/manual/tutorial/build-indexes-in-the-background/#considerations) to keep in mind.
+| sparse() | Create a [sparse](http://docs.mongodb.org/manual/core/index-sparse/) index |
+| expireAfterSeconds() | Creates a [TTL Index](http://docs.mongodb.org/manual/core/index-ttl/) on a date field. |
+
+## Id
+Marks a field in an `@Entity` to be the "_id" field in mongodb.
+
+## Property
+An optional annotation instructing Morphia to persist field in to the document given to mongodb.  By default, the field name is used as 
+the property name.  This can be overridden by passing a String with the new name to the annotation.
+
+## Transient
+Instructs Morphia to ignore this field when converting an entity to a document.  The java keyword `transient` can also be used instead.
+
+## Serialized
+Instructs Morphia to serialize this field.
```

Comment:
This description doesn't make it clear what this is for.  "serialize" must mean something different than "persist", but it's not clear what.